packages/terra-overlay/src/Overlay.jsx

```diff
@@ -53,6 +53,12 @@ const propTypes = {
   zIndex: PropTypes.oneOf(zIndexes),
 };
 
+const KEYCODES = {
```

Comment:
we actually switched over to a node package for these not too long ago. I would go ahead and change that here. https://github.com/cerner/terra-core/commit/45efddb9acfa894b3e7a094b0995fbc7dd1d74a3